install/kubernetes/templates/addons/grafana.yaml.tmpl

```diff
@@ -4,6 +4,8 @@ kind: Service
 metadata:
   name: grafana
   namespace: {ISTIO_NAMESPACE}
+  annotations:
```

Comment:
WARNING: this is the wrong place to edit the templates.

you want to edit: https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/grafana/templates/deployment.yaml

Reply:
yes I know, I commented in #5091 - added the other one above, thanks for the pointer
(while we have 2 I prefer to make both changes to increase the chances that it does work)

Reply:
OK. This sounds reasonable. We desperately need to untangle this mess.